apps/web/src/components/Table/ResponsiveTable/ResponsiveTable.tsx

```diff
@@ -328,8 +328,9 @@ const ResponsiveTable = <TData extends object, TFilters = object>({
     announce(
       intl.formatMessage(
         {
-          defaultMessage: "{count} items found.",
-          id: "5ySBS0",
+          defaultMessage:
+            "{total, plural, =0 {0 results found} =1 {1 result found} other {# result found}}",
+          id: "uZYkk1",
```

Comment:
This isn't working right, it is reading out "total, plural..." for me

Reply:
Oops, forgot the variable name was not total :sweat_smile: 

Fixed in [fix intl variable name](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12429/commits/0f39160a7675acb865c8e052efad89604dea0225)

Reply:
Shouldn't
`other {# result found}}`
Be
`other {# results found}}`
?

Reply:
okay, I tihnk I finally got it :laughing: 

[fix plural other string](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12429/commits/2fd1dc1c354657831406b7cf6fb7edd23b5783d6)